### PR TITLE
remove `anch` macro

### PR DESCRIPTION
**Pull request by Graywolf9**
remove deprecated `anch` macro

### CODE REVIEW

**Review by JuanVqz — APPROVED** [review]
@Graywolf9 Thank you for this! :heart:

**Comment by JuanVqz** [comment]
I'll merge it

File: files/es/learn/accessibility/mobile/index.html [hunk 1]
```diff
@@ -296,7 +296,7 @@ <h4 id="Mantener_los_menús_accesibles.">Mantener los menús accesibles.</h4>
...
 
-<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en {{ anch("Control mechanisms" )}}, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
+<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#Control mechanisms">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```
Comment (yin1999) [inline files/es/learn/accessibility/mobile/index.html:299]: Is it right for this anchor?

```suggestion
<p>Al implementar un menú de este tipo, debe asegurarse de que el control para revelarlo sea accesible mediante los mecanismos de control apropiados (normalmente, toque para dispositivos móviles), como se explica en <a href="#mecanismos_de_control">Control mechanisms</a>, y que el resto La página se aleja o se oculta de alguna manera mientras se accede al menú, para evitar confusiones al navegar.</p>
```

image

Comment (JuanVqz) [reply files/es/learn/accessibility/mobile/index.html:299]: Thank you for pointing this out, could you please open another PR changing this? thanks!

Comment (yin1999) [reply files/es/learn/accessibility/mobile/index.html:299]: I've fixed this in #5792.

Comment (Graywolf9) [reply files/es/learn/accessibility/mobile/index.html:299]: Thank you!

**Pull request merged**